apps/bfd-model/bfd-model-codegen/src/main/java/gov/cms/bfd/model/codegen/RifLayoutsProcessor.java

```diff
@@ -2203,26 +2235,27 @@ private static TypeName selectJavaFieldType(
       boolean isColumnOptional,
       Optional<Integer> columnLength,
       Optional<Integer> columnScale) {
-    if (type == RifColumnType.CHAR
-        && columnLength.orElse(Integer.MAX_VALUE) == 1
-        && !isColumnOptional) return TypeName.CHAR;
-    else if (type == RifColumnType.CHAR
-        && columnLength.orElse(Integer.MAX_VALUE) == 1
-        && isColumnOptional) return ClassName.get(Character.class);
-    else if (type == RifColumnType.CHAR) return ClassName.get(String.class);
-    else if (type == RifColumnType.DATE && columnLength.orElse(0) == 8)
-      return ClassName.get(LocalDate.class);
-    else if (type == RifColumnType.TIMESTAMP && columnLength.orElse(0) == 20)
-      return ClassName.get(Instant.class);
+    if (type == RifColumnType.CHAR) {
+      if (columnLength.orElse(Integer.MAX_VALUE) == 1) {
+        return isColumnOptional ? ClassName.get(Character.class) : TypeName.CHAR;
+      } else {
+        return ClassName.get(String.class);
+      }
+    } else if (type == RifColumnType.DATE) return ClassName.get(LocalDate.class);
+    else if (type == RifColumnType.TIMESTAMP) return ClassName.get(Instant.class);
```

Comment:
Here's some cursed non-bracketed if statements you could have justification to fix 😄

Reply:
yea, already got beat up for fixing some of the others; I basically operated on the premise that if I had to modify a method, I was not going to perpetuate bad coding practices. I'll take a further look at this one, to see it meets my gag criteria.

Reply:
I did add those lines to my gag reflex fixing on non-bracketed code.

Reply:
As an aside, a comment on your request to add tests coverage.....this is where things could get really tricky and really create some scope creep. Where you asked to add test coverage, we run into the proverbial chicken-egg; the RifLayoutsProcessor is part of the bfd-model-codegen which not only doesn't have any tests, it lives only within the constraints of maven to create ORM models based on an Excel spreadsheet. I'd be in favor of a separate ticket to implement some sort of testing, but literally it'll be bootstrapping an entire test system not just adding a test or two to assert some newly added or changed code.

Reply:
Ok, we should capture this as tech debt then.